packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -438,6 +504,16 @@ class TapGestureRecognizer extends BaseTapGestureRecognizer {
   ///  * [GestureDetector.onTap], which exposes this callback.
   GestureTapCallback? onTap;
 
+  /// A pointer that triggered a tap has moved.
+  ///
+  /// This triggers when the pointer moves after the tap gesture has been recognized.
+  ///
```

Comment:
I suggest noting `postAcceptSlopTolerance` here, such as:
```suggestion
  /// This callback is triggered after the tap gesture has been recognized and the pointer starts to move.
  ///
  /// If the pointer moves beyond the `postAcceptSlopTolerance` distance, the tap will be canceled.
  /// To make `onTapMove` more useful, consider setting `postAcceptSlopTolerance` to a larger value,
  /// or to `null` for no limit on movement.
```

Reply:
Ooops marked the wrong line. I meant to replace the 2nd paragraph.